gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -173,8 +173,7 @@
             <dd><span translate>Comment, when used at the start of a line</span></dd>
           </dl>
           <hr />
```

Comment:
```suggestion
          <hr ng-if="!editingDefaults" />
```

The `hr` is not needed when the next line is hidden, so I would hide it too. Otherwise there is a superfluous horizontal line there.